src/cirq_iqm/devices/iqm_device.py

```diff
@@ -212,9 +213,16 @@ def route_circuit(
         for q in measurement_qubits:
             modified_circuit.append(cirq.I(q).with_tags(i_tag))
 
+        if initial_mapping is not None:
+            # reverse the mapping to match the cirq.contrib.routing.greedy convention
+            initial_mapping = {v: k for k, v in initial_mapping.items()}
```

Comment:
Why don't we simply use the  cirq.contrib.routing.greedy convention?

Reply:
I suppose one reason is to allow to re-use the existing `qubit_mapping` values from users' old code?

Reply:
I think then we did not gain much from calling it `initial_mapping`. I would advocate to expose the cirq semantic to the user because the idea was to use cirq functionality instead of `qubit_mapping` to achieve qubit mapping.

Reply:
Otherwise I agree with @q-mat-beu, we should use Cirq conventions everywhere if possible.

Reply:
Ok, convinced.